### PR TITLE
feat(angular): `[(index)]` & ref access

### PR DESCRIPTION
**Pull request by vltansky**

### CODE REVIEW

File: src/pages/angular.mdx [hunk 1]
```diff
@@ -417,3 +482,7 @@ As you see it is really easy to integrate Swiper into your website or app. So he
 - If you have questions about Swiper ask them in <a href="http://stackoverflow.com" target="_blank">StackOverflow</a> or <a href="https://github.com/nolimits4web/swiper/discussions" target="_blank">Swiper Discussions</a>.
 - Create issue on <a href="https://github.com/nolimits4web/swiper/" target="_blank">GitHub</a> if you found a bug.
 - If you are looking for support, we have a private Discord support chat room for <a href="https://www.patreon.com/swiperjs" target="_blank">Swiper Patrons</a>.
+
+```
```
Comment (nolimits4web) [inline src/pages/angular.mdx:486]: What is this? :)

File: src/pages/angular.mdx [hunk 2]
```diff
@@ -229,10 +229,84 @@ export class AppComponent {
...
+  slideNext(){
+    this.swiper.swiperRef.slideNext(100);
+  }
+  slidePrev(){
+    this.swiper.swiperRef.slidePrev(100);
+  }
+}
+```
+
+## Controlling slide index
```
Comment (nolimits4web) [inline src/pages/angular.mdx:274]: I am not sure about this part. We already have 2 ways of getting Swiper instance here: with `ref` and with `onSwiper` event. And with Swiper instance, user can call any api, e.g. `swiper.slideTo($index)`, which has some useful extra parameters.

I already foresee users question (and opened issues):

> What does setting this index do? Does it change slides with transition or not? If with transition, and I want to change it without transition, what should I do? 😄 

I would stick with unified single `swiper.slideTo` method to control this

Comment (vltansky) [reply src/pages/angular.mdx:274]: But this functionality exists from the release of swiper angular. just was undocumented :)
Do you want to remove this functionality from swiper angular too?

Comment (nolimits4web) [reply src/pages/angular.mdx:274]: Yes, we can remove it from swiper angular too :)

**Pull request merged**